lib/beaker/host/unix/exec.rb

```diff
@@ -187,17 +187,22 @@ def ssh_service_restart
   def ssh_permit_user_environment
     case self['platform']
     when /debian|ubuntu|cumulus/
-      exec(Beaker::Command.new("echo '\nPermitUserEnvironment yes' >> /etc/ssh/sshd_config"))
+      exec(Beaker::Command.new("echo 'PermitUserEnvironment yes' | cat - /etc/ssh/sshd_config > /tmp/sshd_config.new"))
```

Comment:
Would probably be better to use a safe tempdir here, via `create_tmpdir_on`. A few reasons I can think of:

 - that should be portable across unixen
 - no rare but wierd race condition issues with processes somehow writing the same file at the same time
 - the path will be unguessable, so no weird "I put a symlink pointing to a privileged file and got you to overwrite it, woohoo!" security gotchas.

If `create_tmpdir_on` is unacceptable (speed?), perhaps we can find a usage of `mktemp` that works on our unixes that would work.